src/ecalc/libraries/libecalc/common/tests/core/models/compressor_modelling/test_simplified_compressor_train.py

```diff
@@ -963,8 +965,8 @@ def test_calculate_enthalpy_change_head_iteration_and_outlet_stream(dry_fluid):
         for stream, pressure, enthalpy_change in zip(inlet_streams, outlet_pressure, enthalpy_change_joule_per_kg)
     ]
 
-    np.testing.assert_allclose(expected_inlet_z, [s.z for s in inlet_streams])
-    np.testing.assert_allclose(expected_outlet_z, [s.z for s in outlet_streams])
-    np.testing.assert_allclose(expected_inlet_kappa, [s.kappa for s in inlet_streams])
-    np.testing.assert_allclose(expected_outlet_kappa, [s.kappa for s in outlet_streams])
-    np.testing.assert_allclose(expected_enthalpy_change, enthalpy_change_joule_per_kg)
+    np.testing.assert_allclose(expected_inlet_z, [s.z for s in inlet_streams], rtol=1e-5)
+    np.testing.assert_allclose(expected_outlet_z, [s.z for s in outlet_streams], rtol=1e-4)
+    np.testing.assert_allclose(expected_inlet_kappa, [s.kappa for s in inlet_streams], rtol=1e-3)
```

Comment:
Had to go down to 1e-3 in relative tolerance for kappa. Not sure why this change is that big

Reply:
any chance of asking Even if he knows? possibly a bugfix etc? It said in documentation how old our Neqsim was...so I guess a few bugfixes and improvements since then...

Reply:
tbf, it is still <1% change, so I think it is quite negligible.

From NeqSimAPI, most changes come from updated database entries. Could try to first update to 2.4.13 (before database change), to see if results are more similar there, but not sure it is worth the work